ipaserver/install/ca.py

```diff
@@ -57,9 +80,14 @@ def install_check(standalone, replica_config, options):
 
     realm_name = options.realm_name
     host_name = options.host_name
-    subject_base = options.subject
 
-    if replica_config is not None:
+    if replica_config is None:
+        ca_subject = options.ca_subject
+        subject_base = options.subject_base
+    else:
+        ca_subject = lookup_ca_subject(options.realm_name)
+        subject_base = replica_config.subject_base
```

Comment:
Rather than duplicating this below in `install_step_0()` and `install_step_1()`, it would be better to store the values in `options._ca_subject` and `options._subject_base` and reuse them in `install_step_0()` and `install_step_1()`.